mfthomps/CyberCIEGE-port-to-Unity3d
# Issue/80 staff avatar assignment

Closes #80.

This creates one of two different "staff" avatars, based on which department the staff is assigned to. Currently just "Tech" and "Security" departments.

Some files were renamed for clarity.

To verify:
- This requires a somewhat recent change to the `SimSecNoUI.exe` located [here](https://nps.box.com/s/xn88owrsvj0fhskmf10coqz83vccn6dd). This file gets copied to your `CyberCIEGE\game\exec` folder
- Run the Introduction Scenario. This has one IT staff (not currently hired). 
- Notice in the Unity Scene hierarchy, a disabled "Staff-Tech--Larry" GameObject. This will be avatar used when the guy is hired and activated (not currently implemented).
- Run the Physical Security scenario. This has one IT staff (hired) and two security guards (not hired).
- Notice in the hierarchy three disabled GameObjects. Two labeled as Security and one labelled as Tech.

Note, you can activate all these disabled staff GameObjects and see which avatar they have been assigned.
Note, actually hiring the Staff won't activate the GameObject in the scene (yet).
